cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction/Pretty.hs

```diff
@@ -0,0 +1,62 @@
+{-# LANGUAGE DisambiguateRecordFields #-}
```

Comment:
Let's not create an entire folder for a module. You can leave `Pretty.hs` in the `Run` dir for now.

Reply:
done